edx/edx-app-ios
# Support organization code filtering when fetching user course enrollments

### Description

[OSPR-1879](https://openedx.atlassian.net/browse/OSPR-1879)

This PR uses the ```ORG_CODE``` set on ```config.yml``` to filter the course enrollments of a user when trying to fetch the videos in the background.

### How to test this PR

Run the app with an ORG_CODE set on config.yml. When fetching videos in the background it should return only course enrollments for that specific organization.

### Reviewers

cc @marcotuts @nedbat
